src/pages/Login/index.js

```diff
@@ -75,21 +121,73 @@ function Login(props) {
   }
 
   return (
-    <div className="ng-login">
-      <div className="logo" />
-      {loading ? (
-        <Loader />
-      ) : (
-        <GoogleLogin
-          clientId={process.env.REACT_APP_GOOGLE_CLIENT_ID}
-          buttonText="Continue with Google"
-          onSuccess={onSignIn}
-          onFailure={onGoogleLoginFail}
-          cookiePolicy={"single_host_origin"}
-          className="google-login"
-        />
-      )}
-    </div>
+    <>
+      <Container className={classes.merakiLogin} maxWidth="lg">
+        <Grid container spacing={2}>
+          <Grid item xs={12} ms={6} md={6}>
+            <Container maxWidth="sm">
+              <Typography
+                sx={{ pt: { xs: "none", md: 24 } }}
+                variant="h4"
+                align={isActive ? "center" : "left"}
+                color="textPrimary"
+                gutterBottom
+              >
+                Embark on your learning journey with Meraki
+              </Typography>
```

Comment:
How can we give sx to `ResponsiveTypography.` It's being made by us right?

Reply:
@Poonam-Singh-Bagh Yes, it's being made by us, but it should still work because the property is being passed to a `Typography` component, which does support `sx`.  The `{...props}` part of the Component definition is used to denote that all of the properties provided to `ResponsiveTypography` should be passed to `Typography`.